device/broadcom/x86_64-broadcom_common/syncd_shm.ini

```diff
@@ -0,0 +1,14 @@
+# This file contains the default memory size for each ASICs in broadcom platform
+# Format: ASIC_ID=Memory_size
+#b77->td3
+b77=256m
```

Comment:
@richardyu-ms what's the previous memory size required for td3 and others?

Reply:
by default it is 64m, it can be configured by platform_env.config